docs/src/css/custom.css

```diff
@@ -35,3 +35,8 @@
   margin: 0 calc(-1 * var(--ifm-pre-padding));
   padding: 0 var(--ifm-pre-padding);
 }
+
+.button.button--green {
```

Comment:
`button--green` might be a bit of a confusing name. Maybe `button--primary` would be better?

Reply:
That would override the primary style that exists. Happy to change it to another name though.
I used the descriptor green as buttons will be default be blue due to docusaurus default styling. I don't want to override the primary stylings at all because the green won't work well on a white background. (Not that we currently have other buttons...)

Reply:
@rcrichton ?

Reply:
Cool, anything that doesn't mention the colour would be good. Just think we should decouple the class from the colour. Not a big deal though.